model/riscv_mem.sail

```diff
@@ -137,17 +137,10 @@ function rvfi_read (addr, width, result) = ()
 $endif
 
 /* NOTE: The rreg effect is due to MMIO. */
```

Comment:
Can also remove this comment. Maybe that was the previous difference that is no longer required?

Reply:
:+1: done